sc-memory/sc-memory/sc_action.cpp

```diff
@@ -291,3 +293,39 @@ std::string ScAction::GetActionClassPrettyString() const
 
   return actionClassName;
 }
+
+ScAddr ScAction::GetExpectedExecutionTimeInMilliseconds() noexcept
+{
+  ScAddr expectedTime;
+  auto const & expectedTimeIterator = m_context->Iterator5(
+      *this,
+      ScType::EdgeDCommonConst,
+      ScType::LinkConst,
+      ScType::EdgeAccessConstPosPerm,
+      ScKeynodes::nrel_expected_execution_time_in_milliseconds);
+  if (expectedTimeIterator->Next())
+    expectedTime = expectedTimeIterator->Get(2);
+  return expectedTime;
+}
+
+void ScAction::SetExpectedExecutionTimeInMilliseconds(sc_uint32 expectedExecutionTimeInMilliseconds) noexcept(false)
+{
+  ScAddr const & expectedTime = GetExpectedExecutionTimeInMilliseconds();
+  if (!m_context->IsElement(expectedTime))
+  {
+    ScAddr const & linkWithTime = m_context->CreateLink();
+    m_context->SetLinkContent(linkWithTime, std::to_string(expectedExecutionTimeInMilliseconds));
```

Comment:
```suggestion
    m_context->SetLinkContent(linkWithTime, expectedExecutionTimeInMilliseconds);
```

Reply:
when I do that test 'InitiateAndWaitActionAndCheckStoredExpectedExecutionTime' fails with 
/home/crafting-bench/work/ostis-web-platform/sc-machine/sc-memory/tests/sc-memory/agents/test_sc_action.cpp:566: Failure
Expected equality of these values:
  actualWaitTimeUint32
    Which is: 0
  waitTime
    Which is: 500
/home/crafting-bench/work/ostis-web-platform/sc-machine/sc-memory/tests/sc-memory/agents/test_sc_action.cpp:567: Failure
Expected equality of these values:
  actualWaitTimeString
    Which is: "\xF4\x1"
  std::to_string(waitTime)
    Which is: "500"